api/lib/domain/usecases/assign-certification-officer-to-jury-session.js

```diff
@@ -23,11 +23,12 @@ module.exports = async function assignCertificationOfficerToJurySession({
   const certificationOfficer = await userRepository.get(certificationOfficerId);
   const { firstName, lastName } = certificationOfficer;
   const certificationOfficerName = `${firstName} ${lastName}`;
+  const finalizedSession = await finalizedSessionRepository.get({ sessionId: integerSessionId });
+
+  finalizedSession.assignCertificationOfficer({ certificationOfficerName });
+
+  await finalizedSessionRepository.save(finalizedSession);
```

Comment:
**Question:** Pourquoi ne pas directement le mettre à jour? J'ai l'impression qu'on ne fait rien de plus

```suggestion
 await finalizedSessionRepository.updateCertificationOfficer({ sessionId: integerSessionId, certificationOfficerName });
```

Reply:
Pour laisser à main à l'objet de gérer ses mutations internes et ne pas permettre une bidouille fine via répo. 

on a ensuite le pattern en mode sandwich comme ça : 

const myEntity = entityRepo.get(id);
myEntity.doSomething() // changement d'état interne encapsulé dans l'objet métier
entityRepo.save(myEntity);

Pour engendrer une modification de l'état du système on est quasi obligé (ou du moins fortement invité) à passer par un objet du domain dont le job est de savoir comment modifier sa structure interne de façon cohérente et valide. 

Pour en savoir plus ça s'appelle la _persistence ignorance_.

Reply:
OK, c'est donc parce qu'on a en plus `this.isPublishable = false;`?